.github/workflows/release-dev.yaml

```diff
@@ -1,53 +1,30 @@
-name: Docker - Release Image (Development)
+name: Release Image (Development)
 
 on:
```

Comment:
A versão mais limpa ficou ótima! Só fiquei na dúvida sobre resumir os eventos de push para `label == 'deploy'`, o que acontece se for realizada uma alteração que não precise de um PR? Ou essa possibilidade não existe? Vale para os arquivos de prod e staging.

Reply:
A ideia é que nem toda alteração precisa de um deploy, dai o deploy para dev é opcional. Em relação a uma alteração que não precise de PR, podemos deixar livre para commitar na main ou requerer que todas as alterações ocorram via PR. Tenho preferência pelo segundo caso, dai a pessoa faz o pr e mergeia

Reply:
Booa! Prefiro a segunda opção também.